### PR TITLE
Updtae snap docs to include new options.

### PR DESCRIPTION
**Pull request by nekhtiari**
Pull Request
============

> Tracked in JIRA by [IN-XXX](https://geoinformationstore.atlassian.net/browse/IN-XXX)
> Paired with @a, @b

### Scope of the PR

Describe the scope of this pull request. Is this a new feature or a bug fix?

### Checklist

- [ ] Checked spelling and grammar
- [ ] Provided code samples where relevant
- [ ] Checked the output of `make html` to ensure new content displays correctly

### CODE REVIEW

File: source/up42-blocks/processing/snap-polarimetric.rst [hunk 1]
```diff
@@ -41,6 +41,9 @@ Supported parameters
   via ``bbox``, or ``contains``, or ``intersect`` for the
   previous data block in the workflow will **first** be clipped and then processed.
   Please note that by **default** this parameter is set to ``false`` which means that the **full scene** will be processed.
+* ``calibration_band``: It applies calibration to provide an imagery in which the pixel values can be directly related to the radar backscatter of the scene. The default option using ``Sigma Nought`` for calibration. Other available options are ``Beta Nought`` and ``Gamma Nought``.
```
Comment (markusuwe) [inline source/up42-blocks/processing/snap-polarimetric.rst]: "to provide imagery"

File: source/up42-blocks/processing/snap-polarimetric.rst [hunk 2]
```diff
@@ -41,6 +41,9 @@ Supported parameters
...
+* ``speckle_filter``: It applies to remove speckle noise from the image. Please note that by **default** this parameter is set to ``true``.
```
Comment (markusuwe) [inline source/up42-blocks/processing/snap-polarimetric.rst]: Would be good to add which speckle filter is used.

File: source/up42-blocks/processing/snap-polarimetric.rst [hunk 3]
```diff
@@ -41,6 +41,9 @@ Supported parameters
...
+* ``linear_to_db``: It converst the pixel intensity values to normalised radar cross-section measured in decibel (dB) units. Please note that by **default** this parameter is set to ``true``.
```
Comment (perusio) [inline source/up42-blocks/processing/snap-polarimetric.rst]: converts the pixel intensity to normalized radar cross-section measured in a decibel (dB) unit.

File: source/up42-blocks/processing/snap-polarimetric.rst [hunk 4]
```diff
@@ -50,49 +53,50 @@ and a Land mask (``land`` set to ``true``).
...
+		"sigma"
+	  ]
+	  }
+	}
 
 Output format
 -------------
 
 AOI.clipped GeoTIFF format. Also in the output file, for each
 polarization a separate band will be associated.
```
Comment (perusio) [inline source/up42-blocks/processing/snap-polarimetric.rst]: Can you explain this better please. I cannot understand what it means. Thank you.

**Pull request merged**